source/Jobbr.Runtime.ForkedExecution/DefaultJsonOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Jobbr.Runtime.ForkedExecution
+{
+    /// <summary>
+    /// Default options for JSON serialization and deserialization.
+    /// These are required because the System.Text.Json doesn't provide global defaults.
+    /// https://github.com/dotnet/runtime/issues/31094 .
```

Comment:
The dot is there because the linter wants to end all comments with a dot, but the link doesn't obviously work if there is a dot in the end. I can move the link above the explanation.

Reply:
That's a bad linting rule IMHO. Not all comments need to be a full sentence, e.g. if you list something or have an URL here. 
But if you like the rule, feel free to leave it as is.

Reply:
We should update the devtools submodule to contain StyleCop so we can configure these globally.